metadata-builders/src/test/java/com/linkedin/metadata/builders/search/BuilderUtilsTest.java

```diff
@@ -24,9 +24,11 @@ public void testGetCorpUserOwners() {
     Owner owner1 = new Owner().setOwner(new CorpuserUrn("t1"));
     Owner owner2 = new Owner().setOwner(new CorpuserUrn("t2"));
     Owner owner3 = new Owner().setOwner(new CorpGroupUrn("t3"));
-    List<Owner> owners = Arrays.asList(owner1, owner2, owner3);
+    Owner owner4 = new Owner().setOwner(new CorpGroupUrn("t-4-t"));
```

Comment:
Maybe change this to `CorpuserUrn` to test both urn types?

Reply:
I added tests or urn and groups. Adjusted the results to match what should be the correct values. These tests do not appear to run when building the project though. All the unit tests report success regardless of the data entered here. The tests are built, just not run.